tests/diagnostics/uninitialized-globals.slang

```diff
@@ -45,4 +45,34 @@ float use_later_writte()
     return writtenLater;
 }
 
+// Should not warn here (glsl layout decoration check)
```

Comment:
I don't think this comment is quite specific enough with "(glsl layout decoration check)".
```suggestion
// Varying inputs never warn
```